app/pages/lab/project-details.cjsx

```diff
@@ -185,6 +185,15 @@ module.exports = React.createClass
             <small className="form-help">Add a brief introduction to get people interested in your project. This will display on your landing page. Note this field renders markdown (<insert link to best markdown tutorial>), so you can add formatting.</small>
           </p>
 
+          <p>
+            <AutoSave resource={@props.project}>
+              <span className="form-label">Workflow Description</span>
+              <br />
+              <textarea className="standard-input full" name="description" value={@props.project.workflow_description} onChange={handleInputChange.bind @props.project} />
```

Comment:
Unless I'm misunderstanding this field, it looks like the `name` attribute is wrong here.